### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by kalbermattenm**
Thanks for your review @rbovard

### CODE REVIEW

File: content/pages/4_roadmap.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Version under development:
 
 * Functionalities status: Frozen
 * Development: In progress
-* Release date: mid-February 2018
+* Release date: 01 April 2018
```
Comment (rbovard) [inline content/pages/4_roadmap.md]: 1st April

File: content/pages/4_roadmap_fr.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Version en développement:
 
 * Statut des fonctionnalités: Figé
 * Développement: En cours
-* Date de sortie: mi-février 2018
+* Date de sortie: 1 avril 2018
```
Comment (rbovard) [inline content/pages/4_roadmap_fr.md]: 1er avril

**Review by rbovard — COMMENTED** [review]
Thanks for the update, LGTM except 2 small details.

Is the features list still up-to-date?

**Comment by kalbermattenm** [comment]
Feature list updated, merging

**Pull request merged**